apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/Playground/DocumentParams/ManualParams/index.tsx

```diff
@@ -23,26 +22,19 @@ type ManualParameterProps = {
 }
 
 function LocalParameterInput({ param, input, setInput }: ManualParameterProps) {
-  const [localInput, setLocalInput] = useState(input.value ?? '')
```

Comment:
unless you need to edit the contents of an input from outside the input itself, there's never a reason for deriving the state of the input into a JS local state